lib/rmt/config.rb

```diff
@@ -6,11 +6,16 @@
 end
 
 Config.load_and_set_settings(
-  '/etc/rmt.conf',
   File.join(__dir__, '../../config/rmt.yml'),
   File.join(__dir__, '../../config/rmt.local.yml')
 )
```

Comment:
Why not simply use: 

```
Config.load_and_set_settings(
  ('/etc/rmt.conf' if File.readable?('/etc/rmt.conf')),
  File.join(__dir__, '../../config/rmt.yml'),
  File.join(__dir__, '../../config/rmt.local.yml')
)
```

that would fix exactly the case that the file is present but not readable I think.

Reply:
That's a good suggestion. I will apply the fix and see how it behaves.

Reply:
Additionally, you could print a warning if the file is there, but not readable: 

```
if File.exist?('/etc/rmt.conf') && !File.readable?('/etc/rmt.conf')
  warn "Config file '/etc/rmt.conf' is not readable by user '#{RMT::CLI::Base.process_user_name}'"
  warn 'Run as root or adjust the permissions.'
end
```

Reply:
I don't think we need that warning. This change should've been there in the first place. What the purpose of the above snippet?

Reply:
It was just a suggestion to help the user understand why the '/etc/rmt.conf' is not getting used in case it cannot get read due to permissions. I'm also fine with the pr as is.